app/src/androidTest/java/com/android/bookswap/ui/books/edit/EditBookTest.kt

```diff
@@ -148,16 +149,31 @@ class EditBookScreenTest {
         .onNodeWithTag(C.Tag.EditBook.language)
         .assertTextContains(sampleBook.language.toString())
   }
-
+  /*
   @Test
```

Comment:
Why is this test commented? If it doesn't work, one cannot simply commented as such, if it is no longer needed as other tests have covered what this test once covered, then please let it be removed rather than staying as a comment.